src/renderer/ducks/validator/selectors.ts

```diff
@@ -25,3 +25,18 @@ export const getValidatorBeaconNodes = createSelector(
     getBeaconDictionary,
     (validator, beacons) => validator?.beaconNodes.map((url) => beacons[url]).filter((beacon) => !!beacon) || [],
 );
+
+export const getBeaconNodesValidators = createSelector(getValidators, getValidatorKeys, (validators, keys) => {
```

Comment:
define return type for selector?

Reply:
you mean define types in generics or on anonymous arrow function?